src/monitoring_service/database.py

```diff
@@ -64,9 +64,11 @@ def get_monitor_request(
     ) -> Optional[MonitorRequest]:
         row = self.conn.execute(
             """
-                SELECT *,
-                    (SELECT chain_id FROM blockchain) As chain_id
-                FROM monitor_request
+                SELECT monitor_request.*,
+                    chain_id,
```

Comment:
Should we reference and/or check that this is the same as blockchain.chain_id somehow. It should be checked during message receival, but...

Reply:
It *is* `blockchain.chain_id`, we don't store that separately in `monitor_request`. Maybe I should prefix the column names with the table name to make that clearer.